<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>750</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>484</width>
          <height>672</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Select Model</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QComboBox" name="comboBox">
              <property name="placeholderText">
               <string>***select***</string>
              </property>
              <item>
               <property name="text">
                <string>***select***</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>human</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mouse</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>rat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>zebrafish</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>fruitFly</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>yeast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>c.elegans</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>frog</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Attributes</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4" colspan="2">
             <widget class="QComboBox" name="comboBox_2">
              <property name="placeholderText">
               <string>***select***</string>
              </property>
              <item>
               <property name="text">
                <string>***select***</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>symbol</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>entrezid</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ensemblid</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Load count data</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Load meta data</string>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Select Result Directory</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Summary</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>131</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Count matrix contain:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>101</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>Editi Meta Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0" colspan="3">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Filter genes with low summized expression across all conditions</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Threeshold</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Normalized</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Apply TMM Normalization</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>Yes</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>No</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Summary</string>
           </property>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>161</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Count matrix after filtering</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>151</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>151</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Update meta data</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_11" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
